en/includes/guides/authentication/saml/federated-idp-initiated-sso.md

```diff
@@ -0,0 +1,219 @@
+# Configure federated IdP-Initiated Single Sign-On (SSO)
+
+Single Sign On (SSO) is an authentication process which allows users to log in once and gain access to an organization's applications without having to authenticate to each individual application.
+
+Federated IdP-initiated SSO extends this capability and enables a user belonging to an external organization to seamlessly access applications within your organization. In this scenario, your organization's Identity Provider (IdP) and the external organization's IdP establish a trust relationship. Therefore, external users can conveniently log in to their own IdP and access applications in another organization without having to create accounts in that organization.
+
+This guide explains how you can implement federated IdP-initiated SSO using {{product_name}}.
+
+## Scenario
+
+The steps below implement the following example scenario.
+
+- Two instances of {{product_name}} running on two different ports (e.g. 9443 and 9444), represent the internal and external IdPs respectively.
+- The first instance (internal IdP) has an application called `travelocity`.
+- A user belonging to the second instance (external IdP) should be able to access `travelocity` without creating an account in the internal IdP.
+
+## Prerequisites
+
+Before you begin, be sure to set up the following:
+
+1. Set up two instances of {{product_name}}.
+
+2. Set a port offset for the external IdP so that it runs on port 9444.
+
+    !!! info
+
+        Learn how to [set an offset]({{base_path}}/references/default-ports/#change-the-offset-for-default-ports).
+
+3. Since there can be cookie issues when the same hostname is configured for both {{product_name}} instances, it is recommended that you configure different hostnames for the servers.
+
+    !!! info
+        Learn how to [change the hostname]({{base_path}}/deploy/change-the-hostname) for a {{product_name}} instance. In this guide, the hostname of the external IdP is changed to `localhost.com`.
+
+## Create the application
+
+The `travelocity.com` application should reside in the internal IdP. Follow the steps below in the internal IdP to register it as an application.
+
+1. On the {{product_name}} Console, go to **Applications**.
```

Comment:
Need to identity which IS console is mentioned (port 9334 or 9444 one). Here 9443 one is used